src/main/java/org/folio/rest/impl/TenantPermsAPI.java

```diff
@@ -341,11 +341,7 @@ private Future<Void> renamePermList(Conn connection, ModuleId moduleId,
             }
           }
           return future;
-        })
-        .compose(cf -> softDeletePermList(permList.values()
-            .stream()
-            .flatMap(List::stream)
-            .collect(Collectors.toList()), connection));
+        });
```

Comment:
This seems like a small change but I don't understand the implications. Are we wanting to not mark perms as deprecated anymore at all? [As I said on the ticket](https://folio-org.atlassian.net/browse/MODPERMS-239?focusedCommentId=241354) I don't understand the concept of a "split permission". Perhaps that's the problem.